src/metrics.cpp

```diff
@@ -86,6 +86,7 @@ boost::synchronized_value<std::list<uint256>> trackedBlocks;
 
 boost::synchronized_value<std::list<std::string>> messageBox;
 boost::synchronized_value<std::string> initMessage;
+boost::synchronized_value<std::string> initMessageAfterLoading;
 bool loaded = false;
```

Comment:
This is unlikely, but now that this variable is detached from a specific string, we should make sure that we do not have unexpected memory issues with memory order. I'd suggest to make this an atomic variable:
```suggestion
std::atomic<bool> loaded = false;
```

Reply:
Agree with your suggestion, modified accordingly.